ISISComputingGroup/ibex_gui
# Ticket 3401: Stop prompting for graph save on close

### Description of work

Stopped graphs opened from OPI from prompting for a save when closed. Not that the same was already done from graphs created by blocks.

### Ticket

#3401

### Acceptance criteria

To test:
* Go to OPI
* Right click on PV and select Process Variable -> Log Plotter
* Close the graph that pops up
* Confirm that it does not prompt you to save

### Unit tests

Very GUI focused, hard to unit test

### System tests

Squish tests: https://github.com/ISISComputingGroup/System_Tests_UI_E4/pull/11

### Documentation
No documentation changed

---

#### Code Review


### Final Steps
